source/installation/modify-system-security.rst

```diff
@@ -73,7 +89,7 @@ Modify System Security
 
    RHEL/CentOS 6
       .. code-block:: sh
-      
```

Comment:
In general you should avoid whitespace changes that are not relevant to the task at hand to keep the diffs clean. In this case you remove a tab which we did not need, so no harm done.